index.html

```diff
@@ -1,6 +1,11 @@
+<!-- Add your HTML markup here -->
+<!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
+<!-- All the images you need are in the 'img' folder -->
+
+
 <!DOCTYPE html>
 <html lang="en">
-<head>
+<header>
```

Comment:
Be careful not to confuse `header` and `head`.

`head` is for the meta information and linking fonts / stylesheets etc.

`header` is part of your page and should include your nav etc.

Reply:
Thanks for the feedback I will implement it accordingly.